gstd/src/common/primitives.rs

```diff
@@ -151,6 +151,12 @@ impl TryFrom<&[u8]> for ActorId {
 )]
 pub struct MessageId([u8; 32]);
 
+impl MessageId {
+    pub(crate) fn inner(&self) -> &[u8; 32] {
+        &self.0
+    }
+}
```

Comment:
Why aren't you satisfied with `AsRef<[u8]>` implementation for `MessageId`?

Reply:
I missed it. Done